docker/build/Dockerfile

```diff
@@ -22,6 +22,15 @@ COPY --from=golangci/golangci-lint:v1.50 /usr/bin/golangci-lint /usr/local/bin/
 RUN wget -O /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64 \
     && chmod +x /usr/local/bin/kind
 
+# adding crd documentation tool here, since docs-build does not have golang base to create binary from go file
+RUN wget -O gen-crd-api-reference-docs_linux_amd64.tar.gz https://github.com/ahmetb/gen-crd-api-reference-docs/archive/refs/tags/v0.3.0.tar.gz \
+    && tar xzfv gen-crd-api-reference-docs_linux_amd64.tar.gz  \
+    && mkdir crd_documentation \
+    && cp -r gen-crd-api-reference-docs-0.3.0/* ./crd_documentation \
+    && cd crd_documentation \
+    && go build \
+    && cp -r * /usr/local/bin/
```

Comment:
Instead of buiding the binary from source, can't we download the prebuilt binary from the release page - https://github.com/ahmetb/gen-crd-api-reference-docs/releases/tag/v0.1.5?

Reply:
The pre-built binary is outdated and there are certain fields, for which the older version is not able to create the documentation. Hence, had to go with the latest version. I had already commented regarding the same on the repo: 
![Screenshot from 2023-01-04 17-45-02](https://user-images.githubusercontent.com/25133864/210553178-e55ad93e-649d-474f-ac56-f5d67f4e53ba.png)
Since, there was no response from them, I chose to go with the version which covers everything for us.

Reply:
I will look into the alternatives

Reply:
@PrasadG193 I tried using some other alternatives, but did not get expected output, most of the output received were incomplete documentation. :/